src/lib/__tests__/DocSearch-test.js

```diff
@@ -394,11 +389,14 @@ describe('DocSearch', () => {
 
   describe('handleSelected', () => {
     it('should change the location', () => {
+      const fakeWindow = { location: { href: null } };
```

Comment:
Could also have written: `global.window = ...` and then the __window didn’t need to be added in the code

Reply:
It still does not works because window or global.window is handled by jsdom no matter what so window.location.href fails completely

Reply:
See https://github.com/jsdom/jsdom/issues/2112

Reply:
any reason why you didn't do it like this: https://github.com/jsdom/jsdom/issues/2112#issuecomment-361195043?

Reply:
It does require the actual implementation code to switch from href to assign, but we could try indeed if that's safe